### ISSUE TITLE
Implement easier self-styling

### ISSUE THREAD

**Issue by Donkeyfish87** [fniessen/org-html-themes#95]
Is it possible to put colors as variables in the styles and then enable users to only change a color palette if they'd like some cosmetic changes to the themes?

**Comment by fniessen** [comment]
Hello Donkeyfish87. That certainly is possible -- and a good idea… Would you like to give it a try?

**Comment by Donkeyfish87** [comment]
Yeah I think I can try, at least within the next couple of weeks.

**Comment by fniessen** [comment]
Perfect!  I'll happily review it, and add it to the core… Just make a pull request when it's done.
